seed/static/seed/js/controllers/data_quality_admin_controller.js

```diff
@@ -380,8 +380,8 @@ angular.module('BE.seed.controller.data_quality_admin', [])
         spinner_utility.show();
         $q.all(promises).then(function () {
           data_quality_service.data_quality_rules($scope.org.id).then(function (update_rules){
```

Comment:
Good catch! I think we'd want to rename the parameter of the callback function to `updated_rules` instead of changing everything to be `update_rules`.